acp/ideas_module.php

```diff
@@ -150,8 +155,44 @@ public function main($id, $mode)
 			}
 		}
 
-		if ($submit)
+		if ($submit || $submit_forum_setup)
 		{
+			if ($submit_forum_setup && (int) $this->config['ideas_forum_id'])
```

Comment:
I would like to see an error message if somebody tries to use this before there is a forum set. Maybe move this up just before `// Do not write values if there are errors`, and give this a nice comment explaining what the heck all this code is going to do :smiley: Then it can set an error[] with the lang key: `ACP_NO_FORUM_SELECTED`